src/components/Trade/hooks/useTradeQuoteService.tsx

```diff
@@ -179,6 +180,16 @@ export const useTradeQuoteService = () => {
   // Update trade quote
   useEffect(() => setValue('quote', tradeQuote), [tradeQuote, setValue])
 
+  // Set slippage if the quote contains a recommended value, else use the default
+  useEffect(
+    () =>
+      setValue(
+        'slippage',
+        tradeQuote?.recommendedSlippage ? tradeQuote.recommendedSlippage : DEFAULT_SLIPPAGE,
```

Comment:
This is where the action happens - if we get a `recommendedSlippage` back on our quote, use it.